package/setup.py

```diff
@@ -558,6 +558,8 @@ def dynamic_author_list():
           'scipy>=1.0.0',
           'matplotlib>=1.5.1',
           'mock',
+           'tidynamics>=1.0.0',
```

Comment:
You don't use `tidynamics` in the code, so this doesn't need to be here.

Reply:
Thanks, i need tidynamics for the tests only, but adding it to package seemed to fix my problem. Ill undo that and try.